Makefile

```diff
@@ -21,46 +21,68 @@
 
 ###################################################################################
 
-# See https://tech.davis-hansson.com/p/make/ for more information
-# about these initial lines.
+# Some modules build their dependencies from variables, we want these to be 
+# evalutated at the last possible moment. For this we use second expansion to 
+# re-evaluate the generate and verify targets a second time.
+#
+# See https://www.gnu.org/software/make/manual/html_node/Secondary-Expansion.html
+.SECONDEXPANSION:
+
+# For details on some of these "prelude" settings, see:
+# https://clarkgrubb.com/makefile-style-guide
 MAKEFLAGS += --warn-undefined-variables --no-builtin-rules
 SHELL := /usr/bin/env bash
-.SHELLFLAGS := -eu -o pipefail -c
+.SHELLFLAGS := -uo pipefail -c
```

Comment:
Changed this to match the config described here: https://github.com/cert-manager/makefile-modules/blob/main/modules/repository-base/base/Makefile